README.rdoc

```diff
@@ -7,6 +7,14 @@ http://delsolr.rubyforge.org
 DelSolr is a light weight ruby wrapper for solr.  It's intention is to expose the full power of solr queries 
 while keeping the interface as ruby-esque as possible.
 
+== Tested Ruby Versions
+Version       Result
```

Comment:
This formatting is not cosmetic. Please use the format you've used elsewhere.

Please note that you can use "View File" in your PR (or the "compare branches UI") on Github to see what your changes look like before you submit the PR for review.

<img width="1083" alt="Screen Shot 2020-10-20 at 9 13 36 AM" src="https://user-images.githubusercontent.com/50927/96590949-a7967200-12b4-11eb-9818-d498af907c88.png">

<img width="603" alt="Screen Shot 2020-10-20 at 9 15 07 AM" src="https://user-images.githubusercontent.com/50927/96591081-d57bb680-12b4-11eb-8e55-8680e00310ed.png">